RecoDataProducts/inc/CosmicTrackSeed.hh

```diff
@@ -31,16 +29,9 @@ namespace mu2e {
     TrkFitFlag	             _status;      // status of processes used to create this seed
     // helixOK: ???, helixConverged: ???, circleInit: ???, Straight: ???, hitsOK: ???
```

Comment:
I think you can safely ignore those bits (not set them).  We could discuss if it's worthwhile to generalize the 'init' bits in this flag.  hitsOK should have the same meaning here (enough hits to perform the fit).

Reply:
I think these are still used by @sophiemiddleton so I'll leaving any changes here up to her